scripts/check_frontend_test_coverage.py

```diff
@@ -79,6 +79,7 @@
     'contribution-opportunities-backend-api.service.ts',
     'contribution-opportunities.service.ts',
     'contributions-and-review.component.ts',
+    'contributor-dashboard-admin-page.component.ts',
```

Comment:
Added it here as the old directive is divided into two separate places and the old one doesn't have the test file.